src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -103,7 +103,6 @@ def format_issue_table(diff_filename: str, issues: List[PullRequestIssue], toggl
                 url=format_comment_url(issue.url, GITHUB_OPEN_PR_BOT_REFERRER),
                 is_handled=str(issue.is_handled),
                 event_count=small_count(issue.event_count),
-                affected_users=small_count(issue.affected_users),
```

Comment:
You don't need to address it in this PR, but can we stop holding/calculating this field now, and save some mem/cpu cycles? Or is it still valuable somewhere else?

Reply:
hmm since we are only testing internally for now, this would still be useful in the case we want to iterate quickly and try it with javascript (for us, react) in the case that there ARE affected users for that language

but i will keep this in mind